popoolahussein/Awesome-books
# Awesome books: style core elements

# Description:
[wireframe]: https://github.com/adejam/javascript-curriculum/blob/main/books/images/awesome_books_core_elements.png
[linter]: https://github.com/popoolahussein/Awesome-books/pull/4/checks
#### In this project, the Awesome Books application has been enhanced with CSS styles to improve its visual appeal and match the provided [wireframe][wireframe].

### General Requirements:
   * Ensured there are no [linter][linter] errors.
   * Followed the correct GitHub flow.
   * Documented work in a professional manner.
   * Adhered to the best practices for HTML & CSS.
   * Added CSS styles to the application to match the provided wireframe.
   * Used plain CSS to style the application.
### Project Requirements
   * Implemented CSS styles to align the application with the overall layout specified in the provided wireframe.
  * Emphasized the overall layout rather than focusing on minor details.
  * Incorporated custom colors, fonts, backgrounds, and other stylistic elements to improve the user interface.
### Changes Made:
  * Header Styling:
    * Improved the appearance of the header to ensure it aligns with the wireframe.
    * Adjusted font size, color, and margin for better visual hierarchy.
  * Book List Styling:
    * Styled the book list section to match the wireframe.
    * Used alternating background colors for book items to enhance readability.
    * Added border and padding to book items for a cleaner look.
  * Form Styling:
    * Styled the form to align with the wireframe.
    * Adjusted input fields and buttons for better alignment and usability.
    * Ensured the add button is positioned correctly at the right side of the form.

### General Layout
  * Ensured the overall layout of the application matches the wireframe.
  * Concentrated on maintaining a consistent look and feel across the application.

### Preview:
  * How to Test:
    * Clone the repository and navigate to the project directory.
    * Open the index.html file in a browser.
    * Verify that the application UI matches the wireframe.
    * Ensure all functionalities (adding, displaying, and removing books) are working as expected.


  * Screenshot of the completed work:

![Screenshot (412)](https://github.com/user-attachments/assets/2b94e6e4-b758-465b-ae24-bb75b3f10b8d)


##### Conclusion:
* This pull request enhances the Awesome Books application by adding CSS styles that improve its visual appeal and align it with the provided wireframe. All changes adhere to the project's requirements and best practices for HTML & CSS.
